2024-05-17 09:23:45,815 INFO    StreamThr :125875 [internal.py:wandb_internal():86] W&B internal server running at pid: 125875, started at: 2024-05-17 09:23:45.814700
2024-05-17 09:23:45,819 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: status
2024-05-17 09:23:45,822 INFO    WriterThread:125875 [datastore.py:open_for_write():87] open: ./wandb/run-20240517_092345-v7bx3xrr/run-v7bx3xrr.wandb
2024-05-17 09:23:45,824 DEBUG   SenderThread:125875 [sender.py:send():379] send: header
2024-05-17 09:23:45,826 DEBUG   SenderThread:125875 [sender.py:send():379] send: run
2024-05-17 09:23:45,830 INFO    SenderThread:125875 [sender.py:_setup_resume():749] checking resume status for None/FACED_test_loo/v7bx3xrr
2024-05-17 09:23:50,829 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:23:55,831 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:00,834 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:05,837 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:10,840 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:15,842 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:20,845 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:25,848 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:30,183 INFO    SenderThread:125875 [dir_watcher.py:__init__():211] watching files in: ./wandb/run-20240517_092345-v7bx3xrr/files
2024-05-17 09:24:30,183 INFO    SenderThread:125875 [sender.py:_start_run_threads():1124] run started: v7bx3xrr with start time 1715909025.814118
2024-05-17 09:24:30,183 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: status_report
2024-05-17 09:24:30,191 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: check_version
2024-05-17 09:24:30,192 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: check_version
2024-05-17 09:24:30,585 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: run_start
2024-05-17 09:24:30,599 DEBUG   HandlerThread:125875 [system_info.py:__init__():26] System info init
2024-05-17 09:24:30,599 DEBUG   HandlerThread:125875 [system_info.py:__init__():41] System info init done
2024-05-17 09:24:30,600 INFO    HandlerThread:125875 [system_monitor.py:start():194] Starting system monitor
2024-05-17 09:24:30,600 INFO    SystemMonitor:125875 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-05-17 09:24:30,600 INFO    HandlerThread:125875 [system_monitor.py:probe():214] Collecting system info
2024-05-17 09:24:30,601 INFO    SystemMonitor:125875 [interfaces.py:start():190] Started cpu monitoring
2024-05-17 09:24:30,602 INFO    SystemMonitor:125875 [interfaces.py:start():190] Started disk monitoring
2024-05-17 09:24:30,604 INFO    SystemMonitor:125875 [interfaces.py:start():190] Started gpu monitoring
2024-05-17 09:24:30,605 INFO    SystemMonitor:125875 [interfaces.py:start():190] Started memory monitoring
2024-05-17 09:24:30,607 INFO    SystemMonitor:125875 [interfaces.py:start():190] Started network monitoring
2024-05-17 09:24:30,627 DEBUG   HandlerThread:125875 [system_info.py:probe():150] Probing system
2024-05-17 09:24:30,630 DEBUG   HandlerThread:125875 [system_info.py:_probe_git():135] Probing git
2024-05-17 09:24:30,638 DEBUG   HandlerThread:125875 [system_info.py:_probe_git():143] Probing git done
2024-05-17 09:24:30,638 DEBUG   HandlerThread:125875 [system_info.py:probe():198] Probing system done
2024-05-17 09:24:30,638 DEBUG   HandlerThread:125875 [system_monitor.py:probe():223] {'os': 'Linux-5.19.0-40-generic-x86_64-with-glibc2.35', 'python': '3.11.3', 'heartbeatAt': '2024-05-17T01:24:30.627584', 'startedAt': '2024-05-17T01:23:45.805623', 'docker': None, 'cuda': None, 'args': ('log.run=2', 'log.proj_name=FACED_test_loo', 'data=FACED', 'model.timeFilterLen=30', 'model.dilation_array=[1,3,6,12]', 'model.seg_att=15', 'model.avgPoolLen=15', 'model.timeSmootherLen=3', 'train.gpus=[0]', 'train.valid_method=loo', 'train.num_workers=8', 'hydra/job_logging=default', 'train.iftest=True'), 'state': 'running', 'program': '/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py', 'codePathLocal': None, 'codePath': 'interp_mlp.py', 'git': {'remote': 'git@github.com:RunminGan1218/CLISA_Hydra.git', 'commit': '7aa52932c017e2d781381336081caf4bf9c8e166'}, 'email': 'shenxk@sustech.edu.cn', 'root': '/home/ncclab/shenxinke/att_model/CLISA_Hydra', 'host': 'Precision5820', 'username': 'ncclab', 'executable': '/home/ncclab/miniconda3/envs/att_model/bin/python', 'cpu_count': 18, 'cpu_count_logical': 36, 'cpu_freq': {'current': 2.3103888888888893, 'min': 1200.0, 'max': 4633.333333333333}, 'cpu_freq_per_core': [{'current': 1.299, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 1.199, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4700.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4700.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4800.0}, {'current': 3.1, 'min': 1200.0, 'max': 4800.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4700.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 1.2, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4700.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4800.0}, {'current': 3.0, 'min': 1200.0, 'max': 4800.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}, {'current': 3.0, 'min': 1200.0, 'max': 4600.0}], 'disk': {'/': {'total': 937.5354804992676, 'used': 874.1124153137207}}, 'gpu': 'NVIDIA GeForce RTX 3080', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3080', 'memory_total': 10737418240}], 'memory': {'total': 251.422119140625}}
2024-05-17 09:24:30,639 INFO    HandlerThread:125875 [system_monitor.py:probe():224] Finished collecting system info
2024-05-17 09:24:30,639 INFO    HandlerThread:125875 [system_monitor.py:probe():227] Publishing system info
2024-05-17 09:24:30,639 DEBUG   HandlerThread:125875 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-05-17 09:24:31,185 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_created():271] file/dir created: ./wandb/run-20240517_092345-v7bx3xrr/files/conda-environment.yaml
2024-05-17 09:24:36,855 DEBUG   HandlerThread:125875 [system_info.py:_save_conda():222] Saving conda packages done
2024-05-17 09:24:36,857 INFO    HandlerThread:125875 [system_monitor.py:probe():229] Finished publishing system info
2024-05-17 09:24:36,858 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: status_report
2024-05-17 09:24:36,858 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:36,859 DEBUG   SenderThread:125875 [sender.py:send():379] send: files
2024-05-17 09:24:36,859 INFO    SenderThread:125875 [sender.py:_save_file():1390] saving file wandb-metadata.json with policy now
2024-05-17 09:24:36,993 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: python_packages
2024-05-17 09:24:36,993 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: stop_status
2024-05-17 09:24:36,994 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: python_packages
2024-05-17 09:24:36,995 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: internal_messages
2024-05-17 09:24:36,995 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: stop_status
2024-05-17 09:24:37,187 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_modified():288] file/dir modified: ./wandb/run-20240517_092345-v7bx3xrr/files/conda-environment.yaml
2024-05-17 09:24:37,188 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_created():271] file/dir created: ./wandb/run-20240517_092345-v7bx3xrr/files/requirements.txt
2024-05-17 09:24:37,188 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_created():271] file/dir created: ./wandb/run-20240517_092345-v7bx3xrr/files/wandb-metadata.json
2024-05-17 09:24:37,321 DEBUG   SenderThread:125875 [sender.py:send():379] send: telemetry
2024-05-17 09:24:37,321 DEBUG   SenderThread:125875 [sender.py:send():379] send: metric
2024-05-17 09:24:37,322 DEBUG   SenderThread:125875 [sender.py:send():379] send: telemetry
2024-05-17 09:24:37,322 DEBUG   SenderThread:125875 [sender.py:send():379] send: metric
2024-05-17 09:24:37,322 WARNING SenderThread:125875 [sender.py:send_metric():1341] Seen metric with glob (shouldn't happen)
2024-05-17 09:24:37,885 INFO    wandb-upload_0:125875 [upload_job.py:push():131] Uploaded file /tmp/tmplqz0fu5rwandb/ppxr93zi-wandb-metadata.json
2024-05-17 09:24:38,188 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_created():271] file/dir created: ./wandb/run-20240517_092345-v7bx3xrr/files/output.log
2024-05-17 09:24:39,293 DEBUG   SenderThread:125875 [sender.py:send():379] send: exit
2024-05-17 09:24:39,294 INFO    SenderThread:125875 [sender.py:send_exit():586] handling exit code: 1
2024-05-17 09:24:39,294 INFO    SenderThread:125875 [sender.py:send_exit():588] handling runtime: 8
2024-05-17 09:24:39,296 INFO    SenderThread:125875 [sender.py:_save_file():1390] saving file wandb-summary.json with policy end
2024-05-17 09:24:39,297 INFO    SenderThread:125875 [sender.py:send_exit():594] send defer
2024-05-17 09:24:39,297 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: defer
2024-05-17 09:24:39,297 INFO    HandlerThread:125875 [handler.py:handle_request_defer():172] handle defer: 0
2024-05-17 09:24:39,298 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: defer
2024-05-17 09:24:39,298 INFO    SenderThread:125875 [sender.py:send_request_defer():610] handle sender defer: 0
2024-05-17 09:24:39,298 INFO    SenderThread:125875 [sender.py:transition_state():614] send defer: 1
2024-05-17 09:24:39,298 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: defer
2024-05-17 09:24:39,298 INFO    HandlerThread:125875 [handler.py:handle_request_defer():172] handle defer: 1
2024-05-17 09:24:39,298 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: defer
2024-05-17 09:24:39,298 INFO    SenderThread:125875 [sender.py:send_request_defer():610] handle sender defer: 1
2024-05-17 09:24:39,298 INFO    SenderThread:125875 [sender.py:transition_state():614] send defer: 2
2024-05-17 09:24:39,298 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: defer
2024-05-17 09:24:39,298 INFO    HandlerThread:125875 [handler.py:handle_request_defer():172] handle defer: 2
2024-05-17 09:24:39,299 INFO    HandlerThread:125875 [system_monitor.py:finish():203] Stopping system monitor
2024-05-17 09:24:39,299 DEBUG   SystemMonitor:125875 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-05-17 09:24:39,300 DEBUG   SystemMonitor:125875 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-05-17 09:24:39,300 INFO    HandlerThread:125875 [interfaces.py:finish():202] Joined cpu monitor
2024-05-17 09:24:39,300 DEBUG   SystemMonitor:125875 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-05-17 09:24:39,300 INFO    HandlerThread:125875 [interfaces.py:finish():202] Joined disk monitor
2024-05-17 09:24:39,364 INFO    HandlerThread:125875 [interfaces.py:finish():202] Joined gpu monitor
2024-05-17 09:24:39,364 INFO    HandlerThread:125875 [interfaces.py:finish():202] Joined memory monitor
2024-05-17 09:24:39,365 INFO    HandlerThread:125875 [interfaces.py:finish():202] Joined network monitor
2024-05-17 09:24:39,365 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: defer
2024-05-17 09:24:39,365 INFO    SenderThread:125875 [sender.py:send_request_defer():610] handle sender defer: 2
2024-05-17 09:24:39,365 INFO    SenderThread:125875 [sender.py:transition_state():614] send defer: 3
2024-05-17 09:24:39,365 DEBUG   SenderThread:125875 [sender.py:send():379] send: stats
2024-05-17 09:24:39,366 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: defer
2024-05-17 09:24:39,366 INFO    HandlerThread:125875 [handler.py:handle_request_defer():172] handle defer: 3
2024-05-17 09:24:39,366 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: defer
2024-05-17 09:24:39,366 INFO    SenderThread:125875 [sender.py:send_request_defer():610] handle sender defer: 3
2024-05-17 09:24:39,366 INFO    SenderThread:125875 [sender.py:transition_state():614] send defer: 4
2024-05-17 09:24:39,367 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: defer
2024-05-17 09:24:39,367 INFO    HandlerThread:125875 [handler.py:handle_request_defer():172] handle defer: 4
2024-05-17 09:24:39,367 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: defer
2024-05-17 09:24:39,367 INFO    SenderThread:125875 [sender.py:send_request_defer():610] handle sender defer: 4
2024-05-17 09:24:39,367 INFO    SenderThread:125875 [sender.py:transition_state():614] send defer: 5
2024-05-17 09:24:39,367 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: defer
2024-05-17 09:24:39,367 INFO    HandlerThread:125875 [handler.py:handle_request_defer():172] handle defer: 5
2024-05-17 09:24:39,367 DEBUG   SenderThread:125875 [sender.py:send():379] send: summary
2024-05-17 09:24:39,368 INFO    SenderThread:125875 [sender.py:_save_file():1390] saving file wandb-summary.json with policy end
2024-05-17 09:24:39,368 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: defer
2024-05-17 09:24:39,368 INFO    SenderThread:125875 [sender.py:send_request_defer():610] handle sender defer: 5
2024-05-17 09:24:39,368 INFO    SenderThread:125875 [sender.py:transition_state():614] send defer: 6
2024-05-17 09:24:39,368 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: defer
2024-05-17 09:24:39,368 INFO    HandlerThread:125875 [handler.py:handle_request_defer():172] handle defer: 6
2024-05-17 09:24:39,369 DEBUG   SenderThread:125875 [sender.py:send_request():406] send_request: defer
2024-05-17 09:24:39,369 INFO    SenderThread:125875 [sender.py:send_request_defer():610] handle sender defer: 6
2024-05-17 09:24:39,374 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: status_report
2024-05-17 09:24:40,190 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_modified():288] file/dir modified: ./wandb/run-20240517_092345-v7bx3xrr/files/output.log
2024-05-17 09:24:40,190 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_created():271] file/dir created: ./wandb/run-20240517_092345-v7bx3xrr/files/wandb-summary.json
2024-05-17 09:24:40,278 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: poll_exit
2024-05-17 09:24:42,191 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_modified():288] file/dir modified: ./wandb/run-20240517_092345-v7bx3xrr/files/output.log
2024-05-17 09:24:45,282 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:50,286 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:24:55,289 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:00,292 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:05,296 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:10,299 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:15,302 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:20,306 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:25,309 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:30,313 DEBUG   HandlerThread:125875 [handler.py:handle_request():146] handle_request: keepalive
2024-05-17 09:25:30,865 WARNING StreamThr :125875 [internal.py:is_dead():414] Internal process exiting, parent pid 125867 disappeared
2024-05-17 09:25:30,865 ERROR   StreamThr :125875 [internal.py:wandb_internal():152] Internal process shutdown.
2024-05-17 09:25:31,304 INFO    WriterThread:125875 [datastore.py:close():296] close: ./wandb/run-20240517_092345-v7bx3xrr/run-v7bx3xrr.wandb
2024-05-17 09:25:31,314 INFO    HandlerThread:125875 [handler.py:finish():866] shutting down handler
2024-05-17 09:25:42,792 INFO    SenderThread:125875 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/connectionpool.py", line 536, in _make_request
    response = conn.getresponse()
               ^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/connection.py", line 461, in getresponse
    httplib_response = super().getresponse()
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/http/client.py", line 1375, in getresponse
    response.begin()
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/http/client.py", line 318, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/http/client.py", line 279, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/socket.py", line 706, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/ssl.py", line 1278, in recv_into
    return self.read(nbytes, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/ssl.py", line 1134, in read
    return self._sslobj.read(len, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TimeoutError: The read operation timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/connectionpool.py", line 844, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/util/retry.py", line 470, in increment
    raise reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/util/util.py", line 39, in reraise
    raise value
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/connectionpool.py", line 790, in urlopen
    response = self._make_request(
               ^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/connectionpool.py", line 538, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/urllib3/connectionpool.py", line 370, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=20)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/wandb/sdk/lib/gql_request.py", line 58, in execute
    request = self.session.post(self.url, **post_args)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/requests/adapters.py", line 532, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=20)
2024-05-17 09:25:43,417 WARNING FileStreamThread:125875 [file_stream.py:request_with_retry():687] requests_with_retry encountered retryable exception: 408 Client Error: Request Timeout for url: https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream. func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7fe49b9f6f90>>, timeout=180.0), args: ('https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"_wandb": {"runtime": 8}}']}, 'wandb-events.jsonl': {'offset': 0, 'content': ['{"system.proc.memory.availableMB": 250094.25, "system.memory": 2.88, "system.proc.memory.rssMB": 543.64, "system.proc.memory.percent": 0.21, "system.network.sent": 455926.0, "system.network.recv": 36737445.4, "system.disk.\\\\.usagePercent": 97.9, "system.disk.\\\\.usageGB": 874.11, "system.disk.in": 0.0, "system.disk.out": 0.02, "system.cpu": 0.09, "system.cpu.0.cpu_percent": 0.66, "system.cpu.1.cpu_percent": 0.02, "system.cpu.2.cpu_percent": 0.02, "system.cpu.3.cpu_percent": 0.1, "system.cpu.4.cpu_percent": 0.06, "system.cpu.5.cpu_percent": 4.14, "system.cpu.6.cpu_percent": 0.58, "system.cpu.7.cpu_percent": 0.0, "system.cpu.8.cpu_percent": 0.02, "system.cpu.9.cpu_percent": 0.14, "system.cpu.10.cpu_percent": 0.28, "system.cpu.11.cpu_percent": 0.0, "system.cpu.12.cpu_percent": 0.52, "system.cpu.13.cpu_percent": 0.72, "system.cpu.14.cpu_percent": 1.74, "system.cpu.15.cpu_percent": 62.28, "system.cpu.16.cpu_percent": 1.12, "system.cpu.17.cpu_percent": 0.24, "system.cpu.18.cpu_percent": 0.2, "system.cpu.19.cpu_percent": 0.0, "system.cpu.20.cpu_percent": 0.22, "system.cpu.21.cpu_percent": 6.2, "system.cpu.22.cpu_percent": 0.44, "system.cpu.23.cpu_percent": 0.0, "system.cpu.24.cpu_percent": 0.0, "system.cpu.25.cpu_percent": 0.2, "system.cpu.26.cpu_percent": 0.0, "system.cpu.27.cpu_percent": 0.0, "system.cpu.28.cpu_percent": 0.0, "system.cpu.29.cpu_percent": 0.0, "system.cpu.30.cpu_percent": 0.0, "system.cpu.31.cpu_percent": 0.1, "system.cpu.32.cpu_percent": 0.0, "system.cpu.33.cpu_percent": 0.0, "system.cpu.34.cpu_percent": 0.0, "system.cpu.35.cpu_percent": 0.0, "system.proc.cpu.threads": 5, "_wandb": true, "_timestamp": 1715909079.301109, "_runtime": 53.486991}']}, 'output.log': {'offset': 0, 'content': ['2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,995][__main__][INFO] - fold:0\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - train_subs:[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122]\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - val_subs:[0]\n', "ERROR 2024-05-17T01:24:41.234896 Error executing job with overrides: ['log.run=2', 'log.proj_name=FACED_test_loo', 'data=FACED', 'model.timeFilterLen=30', 'model.dilation_array=[1,3,6,12]', 'model.seg_att=15', 'model.avgPoolLen=15', 'model.timeSmootherLen=3', 'train.gpus=[0]', 'train.valid_method=loo', 'train.num_workers=8', 'train.iftest=True']\n", 'ERROR 2024-05-17T01:24:41.234896 Traceback (most recent call last):\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 103, in <module>\n', 'ERROR 2024-05-17T01:24:41.234896     interp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/main.py", line 94, in decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896     _run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 394, in _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896     _run_app(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 457, in _run_app\n', 'ERROR 2024-05-17T01:24:41.234896     run_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 223, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     raise ex\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 220, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     return func()\n', 'ERROR 2024-05-17T01:24:41.234896            ^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 458, in <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896     lambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896             ^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/hydra.py", line 132, in run\n', 'ERROR 2024-05-17T01:24:41.234896     _ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896         ^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 260, in return_value\n', 'ERROR 2024-05-17T01:24:41.234896     raise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 186, in run_job\n', 'ERROR 2024-05-17T01:24:41.234896     ret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896                        ^^^^^^^^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 79, in interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896     model_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896     raise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 RuntimeError: Error(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m--------------------------------------------------------------------------------------------------------------------------------------------------------\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 103 <module>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmain.py 94 decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 394 _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_app(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 457 _run_app\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mrun_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 223 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise ex\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 220 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mreturn func()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 458 <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mlambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mhydra.py 132 run\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 260 return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 186 run_job\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 79 interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodel_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodule.py 2152 load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mRuntimeError:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mError(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias". \n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias". \n']}}, 'dropped': 0}}
2024-05-17 09:26:46,233 WARNING FileStreamThread:125875 [file_stream.py:request_with_retry():687] requests_with_retry encountered retryable exception: 408 Client Error: Request Timeout for url: https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream. func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7fe49b9f6f90>>, timeout=180.0), args: ('https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"_wandb": {"runtime": 8}}']}, 'wandb-events.jsonl': {'offset': 0, 'content': ['{"system.proc.memory.availableMB": 250094.25, "system.memory": 2.88, "system.proc.memory.rssMB": 543.64, "system.proc.memory.percent": 0.21, "system.network.sent": 455926.0, "system.network.recv": 36737445.4, "system.disk.\\\\.usagePercent": 97.9, "system.disk.\\\\.usageGB": 874.11, "system.disk.in": 0.0, "system.disk.out": 0.02, "system.cpu": 0.09, "system.cpu.0.cpu_percent": 0.66, "system.cpu.1.cpu_percent": 0.02, "system.cpu.2.cpu_percent": 0.02, "system.cpu.3.cpu_percent": 0.1, "system.cpu.4.cpu_percent": 0.06, "system.cpu.5.cpu_percent": 4.14, "system.cpu.6.cpu_percent": 0.58, "system.cpu.7.cpu_percent": 0.0, "system.cpu.8.cpu_percent": 0.02, "system.cpu.9.cpu_percent": 0.14, "system.cpu.10.cpu_percent": 0.28, "system.cpu.11.cpu_percent": 0.0, "system.cpu.12.cpu_percent": 0.52, "system.cpu.13.cpu_percent": 0.72, "system.cpu.14.cpu_percent": 1.74, "system.cpu.15.cpu_percent": 62.28, "system.cpu.16.cpu_percent": 1.12, "system.cpu.17.cpu_percent": 0.24, "system.cpu.18.cpu_percent": 0.2, "system.cpu.19.cpu_percent": 0.0, "system.cpu.20.cpu_percent": 0.22, "system.cpu.21.cpu_percent": 6.2, "system.cpu.22.cpu_percent": 0.44, "system.cpu.23.cpu_percent": 0.0, "system.cpu.24.cpu_percent": 0.0, "system.cpu.25.cpu_percent": 0.2, "system.cpu.26.cpu_percent": 0.0, "system.cpu.27.cpu_percent": 0.0, "system.cpu.28.cpu_percent": 0.0, "system.cpu.29.cpu_percent": 0.0, "system.cpu.30.cpu_percent": 0.0, "system.cpu.31.cpu_percent": 0.1, "system.cpu.32.cpu_percent": 0.0, "system.cpu.33.cpu_percent": 0.0, "system.cpu.34.cpu_percent": 0.0, "system.cpu.35.cpu_percent": 0.0, "system.proc.cpu.threads": 5, "_wandb": true, "_timestamp": 1715909079.301109, "_runtime": 53.486991}']}, 'output.log': {'offset': 0, 'content': ['2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,995][__main__][INFO] - fold:0\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - train_subs:[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122]\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - val_subs:[0]\n', "ERROR 2024-05-17T01:24:41.234896 Error executing job with overrides: ['log.run=2', 'log.proj_name=FACED_test_loo', 'data=FACED', 'model.timeFilterLen=30', 'model.dilation_array=[1,3,6,12]', 'model.seg_att=15', 'model.avgPoolLen=15', 'model.timeSmootherLen=3', 'train.gpus=[0]', 'train.valid_method=loo', 'train.num_workers=8', 'train.iftest=True']\n", 'ERROR 2024-05-17T01:24:41.234896 Traceback (most recent call last):\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 103, in <module>\n', 'ERROR 2024-05-17T01:24:41.234896     interp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/main.py", line 94, in decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896     _run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 394, in _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896     _run_app(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 457, in _run_app\n', 'ERROR 2024-05-17T01:24:41.234896     run_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 223, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     raise ex\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 220, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     return func()\n', 'ERROR 2024-05-17T01:24:41.234896            ^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 458, in <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896     lambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896             ^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/hydra.py", line 132, in run\n', 'ERROR 2024-05-17T01:24:41.234896     _ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896         ^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 260, in return_value\n', 'ERROR 2024-05-17T01:24:41.234896     raise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 186, in run_job\n', 'ERROR 2024-05-17T01:24:41.234896     ret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896                        ^^^^^^^^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 79, in interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896     model_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896     raise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 RuntimeError: Error(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m--------------------------------------------------------------------------------------------------------------------------------------------------------\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 103 <module>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmain.py 94 decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 394 _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_app(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 457 _run_app\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mrun_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 223 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise ex\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 220 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mreturn func()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 458 <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mlambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mhydra.py 132 run\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 260 return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 186 run_job\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 79 interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodel_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodule.py 2152 load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mRuntimeError:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mError(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias". \n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias". \n']}}, 'dropped': 0}}
2024-05-17 09:27:49,476 INFO    SenderThread:125875 [sender.py:transition_state():614] send defer: 7
2024-05-17 09:27:49,476 INFO    SenderThread:125875 [sender.py:finish():1546] shutting down sender
2024-05-17 09:27:50,343 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_modified():288] file/dir modified: ./wandb/run-20240517_092345-v7bx3xrr/files/config.yaml
2024-05-17 09:27:51,292 WARNING FileStreamThread:125875 [file_stream.py:request_with_retry():687] requests_with_retry encountered retryable exception: 408 Client Error: Request Timeout for url: https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream. func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7fe49b9f6f90>>, timeout=180.0), args: ('https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"_wandb": {"runtime": 8}}']}, 'wandb-events.jsonl': {'offset': 0, 'content': ['{"system.proc.memory.availableMB": 250094.25, "system.memory": 2.88, "system.proc.memory.rssMB": 543.64, "system.proc.memory.percent": 0.21, "system.network.sent": 455926.0, "system.network.recv": 36737445.4, "system.disk.\\\\.usagePercent": 97.9, "system.disk.\\\\.usageGB": 874.11, "system.disk.in": 0.0, "system.disk.out": 0.02, "system.cpu": 0.09, "system.cpu.0.cpu_percent": 0.66, "system.cpu.1.cpu_percent": 0.02, "system.cpu.2.cpu_percent": 0.02, "system.cpu.3.cpu_percent": 0.1, "system.cpu.4.cpu_percent": 0.06, "system.cpu.5.cpu_percent": 4.14, "system.cpu.6.cpu_percent": 0.58, "system.cpu.7.cpu_percent": 0.0, "system.cpu.8.cpu_percent": 0.02, "system.cpu.9.cpu_percent": 0.14, "system.cpu.10.cpu_percent": 0.28, "system.cpu.11.cpu_percent": 0.0, "system.cpu.12.cpu_percent": 0.52, "system.cpu.13.cpu_percent": 0.72, "system.cpu.14.cpu_percent": 1.74, "system.cpu.15.cpu_percent": 62.28, "system.cpu.16.cpu_percent": 1.12, "system.cpu.17.cpu_percent": 0.24, "system.cpu.18.cpu_percent": 0.2, "system.cpu.19.cpu_percent": 0.0, "system.cpu.20.cpu_percent": 0.22, "system.cpu.21.cpu_percent": 6.2, "system.cpu.22.cpu_percent": 0.44, "system.cpu.23.cpu_percent": 0.0, "system.cpu.24.cpu_percent": 0.0, "system.cpu.25.cpu_percent": 0.2, "system.cpu.26.cpu_percent": 0.0, "system.cpu.27.cpu_percent": 0.0, "system.cpu.28.cpu_percent": 0.0, "system.cpu.29.cpu_percent": 0.0, "system.cpu.30.cpu_percent": 0.0, "system.cpu.31.cpu_percent": 0.1, "system.cpu.32.cpu_percent": 0.0, "system.cpu.33.cpu_percent": 0.0, "system.cpu.34.cpu_percent": 0.0, "system.cpu.35.cpu_percent": 0.0, "system.proc.cpu.threads": 5, "_wandb": true, "_timestamp": 1715909079.301109, "_runtime": 53.486991}']}, 'output.log': {'offset': 0, 'content': ['2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,995][__main__][INFO] - fold:0\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - train_subs:[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122]\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - val_subs:[0]\n', "ERROR 2024-05-17T01:24:41.234896 Error executing job with overrides: ['log.run=2', 'log.proj_name=FACED_test_loo', 'data=FACED', 'model.timeFilterLen=30', 'model.dilation_array=[1,3,6,12]', 'model.seg_att=15', 'model.avgPoolLen=15', 'model.timeSmootherLen=3', 'train.gpus=[0]', 'train.valid_method=loo', 'train.num_workers=8', 'train.iftest=True']\n", 'ERROR 2024-05-17T01:24:41.234896 Traceback (most recent call last):\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 103, in <module>\n', 'ERROR 2024-05-17T01:24:41.234896     interp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/main.py", line 94, in decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896     _run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 394, in _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896     _run_app(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 457, in _run_app\n', 'ERROR 2024-05-17T01:24:41.234896     run_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 223, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     raise ex\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 220, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     return func()\n', 'ERROR 2024-05-17T01:24:41.234896            ^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 458, in <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896     lambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896             ^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/hydra.py", line 132, in run\n', 'ERROR 2024-05-17T01:24:41.234896     _ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896         ^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 260, in return_value\n', 'ERROR 2024-05-17T01:24:41.234896     raise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 186, in run_job\n', 'ERROR 2024-05-17T01:24:41.234896     ret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896                        ^^^^^^^^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 79, in interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896     model_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896     raise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 RuntimeError: Error(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m--------------------------------------------------------------------------------------------------------------------------------------------------------\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 103 <module>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmain.py 94 decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 394 _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_app(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 457 _run_app\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mrun_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 223 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise ex\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 220 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mreturn func()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 458 <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mlambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mhydra.py 132 run\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 260 return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 186 run_job\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 79 interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodel_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodule.py 2152 load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mRuntimeError:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mError(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias". \n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias". \n']}}, 'dropped': 0}}
2024-05-17 09:27:53,383 INFO    SenderThread:125875 [dir_watcher.py:finish():358] shutting down directory watcher
2024-05-17 09:27:54,345 INFO    Thread-12 :125875 [dir_watcher.py:_on_file_modified():288] file/dir modified: ./wandb/run-20240517_092345-v7bx3xrr/files/output.log
2024-05-17 09:27:54,346 INFO    SenderThread:125875 [dir_watcher.py:finish():388] scan: ./wandb/run-20240517_092345-v7bx3xrr/files
2024-05-17 09:27:54,347 INFO    SenderThread:125875 [dir_watcher.py:finish():402] scan save: ./wandb/run-20240517_092345-v7bx3xrr/files/wandb-metadata.json wandb-metadata.json
2024-05-17 09:27:54,347 INFO    SenderThread:125875 [dir_watcher.py:finish():402] scan save: ./wandb/run-20240517_092345-v7bx3xrr/files/requirements.txt requirements.txt
2024-05-17 09:27:54,347 INFO    SenderThread:125875 [dir_watcher.py:finish():402] scan save: ./wandb/run-20240517_092345-v7bx3xrr/files/config.yaml config.yaml
2024-05-17 09:27:54,347 INFO    SenderThread:125875 [dir_watcher.py:finish():402] scan save: ./wandb/run-20240517_092345-v7bx3xrr/files/output.log output.log
2024-05-17 09:27:54,348 INFO    SenderThread:125875 [dir_watcher.py:finish():402] scan save: ./wandb/run-20240517_092345-v7bx3xrr/files/wandb-summary.json wandb-summary.json
2024-05-17 09:27:54,356 INFO    SenderThread:125875 [dir_watcher.py:finish():402] scan save: ./wandb/run-20240517_092345-v7bx3xrr/files/conda-environment.yaml conda-environment.yaml
2024-05-17 09:27:54,356 INFO    SenderThread:125875 [file_pusher.py:finish():172] shutting down file pusher
2024-05-17 09:27:54,356 INFO    SenderThread:125875 [file_pusher.py:join():178] waiting for file pusher
2024-05-17 09:27:54,910 INFO    wandb-upload_1:125875 [upload_job.py:push():131] Uploaded file ./wandb/run-20240517_092345-v7bx3xrr/files/config.yaml
2024-05-17 09:27:55,192 INFO    wandb-upload_0:125875 [upload_job.py:push():131] Uploaded file ./wandb/run-20240517_092345-v7bx3xrr/files/requirements.txt
2024-05-17 09:27:55,247 INFO    wandb-upload_3:125875 [upload_job.py:push():131] Uploaded file ./wandb/run-20240517_092345-v7bx3xrr/files/wandb-summary.json
2024-05-17 09:27:55,355 INFO    wandb-upload_2:125875 [upload_job.py:push():131] Uploaded file ./wandb/run-20240517_092345-v7bx3xrr/files/output.log
2024-05-17 09:27:55,413 INFO    wandb-upload_4:125875 [upload_job.py:push():131] Uploaded file ./wandb/run-20240517_092345-v7bx3xrr/files/conda-environment.yaml
2024-05-17 09:27:55,858 INFO    SenderThread:125875 [file_stream.py:finish():614] file stream finish called
2024-05-17 09:29:01,263 WARNING FileStreamThread:125875 [file_stream.py:request_with_retry():687] requests_with_retry encountered retryable exception: 408 Client Error: Request Timeout for url: https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream. func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7fe49b9f6f90>>, timeout=180.0), args: ('https://api.wandb.ai/files/sustech-shenxk/FACED_test_loo/v7bx3xrr/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"_wandb": {"runtime": 8}}']}, 'wandb-events.jsonl': {'offset': 0, 'content': ['{"system.proc.memory.availableMB": 250094.25, "system.memory": 2.88, "system.proc.memory.rssMB": 543.64, "system.proc.memory.percent": 0.21, "system.network.sent": 455926.0, "system.network.recv": 36737445.4, "system.disk.\\\\.usagePercent": 97.9, "system.disk.\\\\.usageGB": 874.11, "system.disk.in": 0.0, "system.disk.out": 0.02, "system.cpu": 0.09, "system.cpu.0.cpu_percent": 0.66, "system.cpu.1.cpu_percent": 0.02, "system.cpu.2.cpu_percent": 0.02, "system.cpu.3.cpu_percent": 0.1, "system.cpu.4.cpu_percent": 0.06, "system.cpu.5.cpu_percent": 4.14, "system.cpu.6.cpu_percent": 0.58, "system.cpu.7.cpu_percent": 0.0, "system.cpu.8.cpu_percent": 0.02, "system.cpu.9.cpu_percent": 0.14, "system.cpu.10.cpu_percent": 0.28, "system.cpu.11.cpu_percent": 0.0, "system.cpu.12.cpu_percent": 0.52, "system.cpu.13.cpu_percent": 0.72, "system.cpu.14.cpu_percent": 1.74, "system.cpu.15.cpu_percent": 62.28, "system.cpu.16.cpu_percent": 1.12, "system.cpu.17.cpu_percent": 0.24, "system.cpu.18.cpu_percent": 0.2, "system.cpu.19.cpu_percent": 0.0, "system.cpu.20.cpu_percent": 0.22, "system.cpu.21.cpu_percent": 6.2, "system.cpu.22.cpu_percent": 0.44, "system.cpu.23.cpu_percent": 0.0, "system.cpu.24.cpu_percent": 0.0, "system.cpu.25.cpu_percent": 0.2, "system.cpu.26.cpu_percent": 0.0, "system.cpu.27.cpu_percent": 0.0, "system.cpu.28.cpu_percent": 0.0, "system.cpu.29.cpu_percent": 0.0, "system.cpu.30.cpu_percent": 0.0, "system.cpu.31.cpu_percent": 0.1, "system.cpu.32.cpu_percent": 0.0, "system.cpu.33.cpu_percent": 0.0, "system.cpu.34.cpu_percent": 0.0, "system.cpu.35.cpu_percent": 0.0, "system.proc.cpu.threads": 5, "_wandb": true, "_timestamp": 1715909079.301109, "_runtime": 53.486991}']}, 'output.log': {'offset': 0, 'content': ['2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,995][__main__][INFO] - fold:0\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - train_subs:[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122]\n', '2024-05-17T01:24:39.326515 [2024-05-17 09:24:36,996][__main__][INFO] - val_subs:[0]\n', "ERROR 2024-05-17T01:24:41.234896 Error executing job with overrides: ['log.run=2', 'log.proj_name=FACED_test_loo', 'data=FACED', 'model.timeFilterLen=30', 'model.dilation_array=[1,3,6,12]', 'model.seg_att=15', 'model.avgPoolLen=15', 'model.timeSmootherLen=3', 'train.gpus=[0]', 'train.valid_method=loo', 'train.num_workers=8', 'train.iftest=True']\n", 'ERROR 2024-05-17T01:24:41.234896 Traceback (most recent call last):\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 103, in <module>\n', 'ERROR 2024-05-17T01:24:41.234896     interp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/main.py", line 94, in decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896     _run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 394, in _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896     _run_app(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 457, in _run_app\n', 'ERROR 2024-05-17T01:24:41.234896     run_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 223, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     raise ex\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 220, in run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896     return func()\n', 'ERROR 2024-05-17T01:24:41.234896            ^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/utils.py", line 458, in <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896     lambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896             ^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/_internal/hydra.py", line 132, in run\n', 'ERROR 2024-05-17T01:24:41.234896     _ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896         ^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 260, in return_value\n', 'ERROR 2024-05-17T01:24:41.234896     raise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/hydra/core/utils.py", line 186, in run_job\n', 'ERROR 2024-05-17T01:24:41.234896     ret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896                        ^^^^^^^^^^^^^^^^^^^^^^^\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/shenxinke/att_model/CLISA_Hydra/interp_mlp.py", line 79, in interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896     model_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896   File "/home/ncclab/miniconda3/envs/att_model/lib/python3.11/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896     raise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 RuntimeError: Error(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias".\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m--------------------------------------------------------------------------------------------------------------------------------------------------------\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 103 <module>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmain.py 94 decorated_main\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_hydra(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 394 _run_hydra\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_run_app(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 457 _run_app\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mrun_and_report(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 223 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise ex\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 220 run_and_report\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mreturn func()\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 458 <lambda>\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mlambda: hydra.run(\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mhydra.py 132 run\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m_ = ret.return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 260 return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise self._return_value\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mutils.py 186 run_job\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mret.return_value = task_function(task_cfg)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1minterp_mlp.py 79 interp_mlp\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodel_mlp.load_state_dict(model_weights)\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mmodule.py 2152 load_state_dict\n', "ERROR 2024-05-17T01:24:41.234896 \x1b[1mraise RuntimeError('Error(s) in loading state_dict for {}:\\n\\t{}'.format(\n", 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mRuntimeError:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1mError(s) in loading state_dict for simpleNN3:\n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tMissing key(s) in state_dict: "fc1.weight", "fc1.bias", "bn1.running_mean", "bn1.running_var", "fc2.weight", "fc2.bias", "bn2.running_mean", "bn2.running_var", "fc3.weight", "fc3.bias". \n', 'ERROR 2024-05-17T01:24:41.234896 \x1b[1m\tUnexpected key(s) in state_dict: "model.fc1.weight", "model.fc1.bias", "model.bn1.running_mean", "model.bn1.running_var", "model.bn1.num_batches_tracked", "model.fc2.weight", "model.fc2.bias", "model.bn2.running_mean", "model.bn2.running_var", "model.bn2.num_batches_tracked", "model.fc3.weight", "model.fc3.bias". \n']}}, 'dropped': 0}}
2024-05-17 09:29:20,223 INFO    SenderThread:125875 [file_stream.py:finish():618] file stream finish is done
